ATS Routes/Conventional - Non-UK/Lower/B24.txt

```diff
@@ -1,5 +1,4 @@
-MTD   MTD   NEBRU NEBRU
+EGOZE EGOZE NEBRU NEBRU
 NEBRU NEBRU DIPER DIPER
 DIPER DIPER BNE   BNE
-BNE   BNE   ING   ING
-ING   ING   B24FIR B24FIR
```

Comment:
I don't think B24FIR is used anywhere else, so could probably be removed from Navaids/Fixes_Non-UK/FIXES_Other.txt ?